espresso/src/main/resources/deviceInfo.json

```diff
@@ -0,0 +1,374 @@
+    "statusBarHeight": 115,
+    "navBarHeight": 132,
+    "deviceHeight": 2400
+  },
+  "oneplus 10t 5g": {
+    "statusBarHeight": 107,
+    "navBarHeight": 132,
+    "deviceHeight": 2412
+  },
+  "samsung galaxy a33 5g": {
+    "statusBarHeight": 80,
+    "navBarHeight": 135,
+    "deviceHeight": 2400
+  },
+  "oppo oppo reno7": {
+    "statusBarHeight": 107,
+    "navBarHeight": 132,
+    "deviceHeight": 2400
+  },
+  "oneplus nord ce 2 lite 5g": {
+    "statusBarHeight": 110,
+    "navBarHeight": 132,
+    "deviceHeight": 2412
+  },
+  "redmi note 8": {
+    "statusBarHeight": 80
+  },
+  "google pixel 6a": {
+    "statusBarHeight": 132,
+    "navBarHeight": 63,
+    "deviceHeight": 2400
+  },
+  "redmi note 10 pro": {
+    "statusBarHeight": 93,
+    "navBarHeight": 130,
+    "deviceHeight": 2400
+  },
+  "oppo cph2239": {
+    "statusBarHeight": 72,
+    "navBarHeight": 88,
+    "deviceHeight": 1600
+  },
+  "samsung galaxy m31": {
+    "statusBarHeight": 83,
+    "navBarHeight": 126,
+    "deviceHeight": 2340
+  },
+  "oneplus9r": {
+    "statusBarHeight": 103,
+    "navBarHeight": 132,
+    "deviceHeight": 2400
+  },
+  "oneplus 8": {
+    "statusBarHeight": 104
+  },
+  "samsung galaxy s22+": {
+    "statusBarHeight": 74,
+    "navBarHeight": 135,
+    "deviceHeight": 2340
+  },
+  "samsung galaxy a13": {
+    "statusBarHeight": 64,
+    "navBarHeight": 135,
+    "deviceHeight": 2408
+  },
+  "samsung galaxy m32 5g": {
+    "statusBarHeight": 45,
+    "navBarHeight": 90,
+    "deviceHeight": 1600
+  },
+  "samsung galaxy s21+ 5g": {
+    "statusBarHeight": 75,
+    "navBarHeight": 135,
+    "deviceHeight": 2400
+  },
+  "redmi note 10": {
+    "statusBarHeight": 93,
+    "navBarHeight": 130,
+    "deviceHeight": 2400
+  },
+  "redmi note 12 pro + 5g": {
+    "statusBarHeight": 94,
+    "navBarHeight": 130,
+    "deviceHeight": 2400
+  },
+  "xiaomi redmi  note  7 pro": {
+    "statusBarHeight": 80
+  },
+  "redmi note 9 pro max": {
+    "statusBarHeight": 102
+  },
+  "google pixel 2": {
+    "statusBarHeight": 63
+  },
+  "google pixel 4 xl": {
+    "statusBarHeight": 98
+  },
+  "motorola moto g(9) play": {
+    "statusBarHeight": 54
+  },
+  "samsung galaxy s20+ 5g": {
+    "statusBarHeight": 73
+  },
+  "samsung galaxy z fold4": {
+    "statusBarHeight": 87,
+    "navBarHeight": 126,
+    "deviceHeight": 2316
+  },
+  "samsung galaxy a52": {
+    "statusBarHeight": 88,
+    "navBarHeight": 126,
+    "deviceHeight": 2400
+  },
+  "vivo v2050": {
+    "statusBarHeight": 88,
+    "navBarHeight": 116,
+    "deviceHeight": 2404
+  },
+  "motorola moto g(7) play": {
+    "statusBarHeight": 72
+  },
+  "samsung galaxy a11": {
+    "statusBarHeight": 65
+  },
+  "samsung galaxy m52 5g": {
+    "statusBarHeight": 90,
+    "navBarHeight": 126,
+    "deviceHeight": 2400
+  },
+  "samsung galaxy tab s7": {
+    "statusBarHeight": 51
+  },
+  "vivo v2111": {
+    "statusBarHeight": 60,
+    "navBarHeight": 79,
+    "deviceHeight": 1600
+  }
+}
```

Comment:
new line

Reply:
my linter is removing extra line after saving 😿